modules/govuk/manifests/apps/email_alert_api/checks.pp

```diff
@@ -4,8 +4,20 @@
 #
 class govuk::apps::email_alert_api::checks(
   $ensure = present,
+  $queue_resources = {
+    'delivery_immediate_high'     => { latency_warning  => '2.5', latency_critical => '5' },
+    'delivery_immediate'          => { latency_warning  => '2.5', latency_critical => '5' },
+    'delivery_digest'             => { latency_warning  => '90', latency_critical => '60' },
+    'email_generation_immediate'  => { },
+    'process_and_generate_emails' => { },
+    'default'                     => { },
+    'email_generation_digest'     => { },
+    'cleanup'                     => { },
+  },
```

Comment:
Does this need to be a parameter (for example, does it vary per environment), or could this look something like this [1]?

1: https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/apps/email_alert_api.pp#L204-L219